indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -0,0 +1,91 @@
+                                   data-href="{{ url_for('tracks.edit_track', track) }}"
+                                   data-title="{% trans title=track.title %}Edit {{ title }}{% endtrans %}"
+                                   data-reload-after
+                                   data-ajax-dialog></a>
+                                <a href="#" class="i-button icon-remove js-delete-item"
+                                   title="{% trans %}Delete track{% endtrans %}"
+                                   data-href="{{ url_for('tracks.delete_track', track) }}"
+                                   data-confirm="{% trans %}Are you sure you want to delete this track?{% endtrans %}"
+                                   data-title="{% trans %}Delete track{% endtrans %}"
+                                   data-method="DELETE"
+                                   data-update="#track-list"></a>
+                            </div>
+                        </div>
+                        <span class="i-box-description">{{ track.code }}</span>
+                    </div>
+                    {% if track.description %}
+                        <div class="i-box-content">
+                            {{ track.description }}
+                        </div>
+                    {% endif %}
+                </li>
+            {% endfor %}
+        </ul>
+    {% endif %}
+
+    <script>
+        var heightLimit = 50;
+        $(document).ready(function() {
+            $('.i-box-content').each(function() {
+            var innerHeight =$(this).innerHeight()-$(this).css('paddingTop').replace('px','')- $(this).css('paddingBottom').replace('px','');
+             if( innerHeight > heightLimit){
+                 $(this).addClass('gradient');
+                 $(this).click(function() {
+                     $(this).parent().find('.icon-edit').click();
+                 });
+             }
+            });
+
+            $('.i-box').last().css('margin-bottom', '150px');
+        });
+
+        $('#track-list').sortable({
+            axis: 'y',
+            containment: 'parent',
+            cursor: 'move',
+            distance: 2,
+            handle: '.ui-i-box-sortable-handle',
+            items: '> li.track-row',
+            tolerance: 'pointer',
+            forcePlaceholderSize: true,
+            update: function(e) {
+                var $elem = $('#track-list');
+                var sortedList = $elem.find('li.track-row').map(function(i, elem) {
+                    return $(elem).data('id');
+                }).get();
+
+                $.ajax({
+                    url: $elem.data('url'),
+                    method: 'POST',
+                    contentType: 'application/json',
+                    data: JSON.stringify({'sort_order': sortedList}),
+                    complete: IndicoUI.Dialogs.Util.progress(),
+                    error: handleAjaxError
+                });
+
+                $('.i-box').each(function() {
```

Comment:
I'm almost positive this can be done in CSS only. We can sit together and take a look into it.